lib/simplecov/version.rb

```diff
@@ -1,22 +1,25 @@
 module SimpleCov
-  VERSION = "0.11.1"
-  def VERSION.to_a
+  version = "0.11.1"
```

Comment:
There's a new cop that requires all literals assigned to constants to be frozen. Because the method definitions that follow modify the `VERSION` string, I had to do these gymnastics to make everything happy.

Perhaps we should create a `Version` module and make those methods class methods of that module?

I'm cool with going with whatever approach you all prefer!